moveit_ros/hybrid_planning/CMakeLists.txt

```
@@ -0,0 +1,88 @@
+cmake_minimum_required(VERSION 3.10.2)
+project(moveit_hybrid_planning)
+
+# Common cmake code applied to all moveit packages
+find_package(moveit_common REQUIRED)
+moveit_package()
+
+# find dependencies
+find_package(ament_cmake REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(rclcpp_components REQUIRED)
+find_package(rclcpp_action REQUIRED)
+find_package(moveit_msgs REQUIRED)
+find_package(moveit_ros_planning REQUIRED)
+find_package(moveit_core REQUIRED)
+find_package(std_srvs REQUIRED)
+find_package(std_msgs REQUIRED)
+find_package(tf2_ros REQUIRED)
+find_package(trajectory_msgs REQUIRED)
+find_package(control_toolbox REQUIRED)
+
+set(LIBRARIES
+  moveit_hybrid_planning_manager
+  moveit_local_planner_component
+  moveit_global_planner_component
+  planner_logic_interface
+  single_plan_execution_plugin
+  trajectory_operator_interface
+  local_constraint_solver_interface
+  simple_sampler_plugin
+  single_plan_execution_plugin
+  replan_invalidated_trajectory_plugin
+  forward_trajectory_plugin
+)
+
+set(THIS_PACKAGE_INCLUDE_DEPENDS
+  rclcpp
+  rclcpp_components
+  rclcpp_action
+  moveit_core
+  moveit_ros_planning
+  moveit_msgs
+  std_msgs
+  std_srvs
+  tf2_ros
+  trajectory_msgs
+  control_toolbox
+)
+
+set(THIS_PACKAGE_INCLUDE_DIRS
+  global_planner/include
+  hybrid_planning_manager/hybrid_planning_manager_component/include
+  hybrid_planning_manager/planner_logic_plugins/include
+  local_planner/local_planner_component/include
+  local_planner/trajectory_operator_plugins/include
+  local_planner/local_constraint_solver_plugins/include
```

```suggestion
  global_planner/include/
  hybrid_planning_manager/hybrid_planning_manager_component/include/
  hybrid_planning_manager/planner_logic_plugins/include/
  local_planner/local_planner_component/include/
  local_planner/trajectory_operator_plugins/include/
  local_planner/local_constraint_solver_plugins/include/
```

Without the trailing slash, these are installed at `install/moveit_hybrid_planning/include/include/moveit/.....` (notice the extra include) and other packages cannot find them without adding an include in the path